### PR TITLE
Admin Antag Hud

### PR DESCRIPTION
**Pull request by Chronitonity**
Allows admins to toggle an antag hud (Which only works for revs/ops/shadowlings/cult at the moment, the original TG works this way and I'm not smart enough or motivated enough to figure that out right now.) that shows the little symbol above an antags head.

Logs in asay when an admin toggles it. So don't do it while you play or I'll personally wake up from my hell sleep and blow up.

### CODE REVIEW

**Comment by TDcoolguy300** [comment]
:+1:

**Comment by Weisyl** [comment]
Noice.

**Comment by Lawligagger** [comment]
Work on it a bit more to have it detect the reset of the antags.

**Comment by Chronitonity** [comment]
Not gonna be possible, if someone else wants to work on the framework, they can, but theres a 0% chance of me getting sprites, reworking hud code, and fixing it just to work with this.

**Comment by TDcoolguy300** [comment]
Rip

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -928,4 +928,23 @@ var/list/datum/outfit/custom_outfits = list() //Admin created outfits
...
+	usr << browse(dat, "window=dressup;size=550x600")
+
+/client/proc/toggle_antag_hud()
+	set category = "Admin"
+	set name = "Toggle AntagHUD"
+	set desc = "Toggles the Admin AntagHUD"
+
+	if(!holder) return
+
+	var/datum/atom_hud/magical = huds[ANTAG_HUD_WIZ]
```
Comment (Carbonhello) [inline code/modules/admin/verbs/randomverbs.dm:940]: Why is this separate from the for loop 2 linea after this,i don't get it

**Comment by danhibiki** [comment]
Good

**Comment by Nebyab** [comment]
Yeah sure. I'd like to see it expanded for all antags, but this is fine as is. We can always add the rest at a later date.

**Pull request merged**